### ISSUE TITLE
date parse error when running docker all-facts

### ISSUE THREAD

**Issue by mkinney** [Fizzadar/pyinfra#288]
**Describe the bug**
Error when running `all-facts` on docker centos.

Get this error:

```
--> An unexpected exception occurred:

  File "/Users/mikekinney/python/mkinney_pyinfra/venv/lib/python3.7/site-packages/python_dateutil-2.8.1-py3.7.egg/dateutil/parser/_parser.py", line 649, in parse
    raise ParserError("Unknown string format: %s", timestr)
dateutil.parser._parser.ParserError: Unknown string format: %date%-%time%
```

**To Reproduce**
pyinfra @docker/ubuntu,@docker/centos all-facts

**Expected behavior**
No errors would be nice. :-)

**Meta**
+ Include output of `pyinfra --support`.

```
mikekinney@sweet mkinney_pyinfra % pyinfra --support
--> Support information:

    If you are having issues with pyinfra or wish to make feature requests, please
    check out the GitHub issues at https://github.com/Fizzadar/pyinfra/issues .
    When adding an issue, be sure to include the following:

    System: Darwin
      Platform: Darwin-19.3.0-x86_64-i386-64bit
      Release: 19.3.0
      Machine: x86_64
    pyinfra: v0.14.1
    Executable: /Users/mikekinney/python/mkinney_pyinfra/venv/bin/pyinfra
    Python: 3.7.6 (CPython, Clang 11.0.0 (clang-1100.0.33.16))
```

+ How was pyinfra installed (source/pip)?
+ Include pyinfra-debug.log (if one was created)

+ Consider including output with `-vv` and `--debug`.

with both of those flags:

```
    [@docker/centos] docker build complete, image ID: 67fb0c2ba1c5
--> An unexpected exception occurred:

  File "/Users/mikekinney/python/mkinney_pyinfra/venv/lib/python3.7/site-packages/python_dateutil-2.8.1-py3.7.egg/dateutil/parser/_parser.py", line 649, in parse
    raise ParserError("Unknown string format: %s", timestr)
dateutil.parser._parser.ParserError: Unknown string format: %date%-%time%
    [pyinfra_cli.exceptions]   File "/Users/mikekinney/python/mkinney_pyinfra/pyinfra_cli/main.py", line 222, in main
    _main(*args, **kwargs)
  File "/Users/mikekinney/python/mkinney_pyinfra/pyinfra_cli/main.py", line 514, in _main
    args=args,
  File "/Users/mikekinney/python/mkinney_pyinfra/pyinfra/api/facts.py", line 232, in get_facts
    data = fact.process(stdout)
  File "/Users/mikekinney/python/mkinney_pyinfra/pyinfra/facts/windows.py", line 193, in process
    return parse_date(new_output)
  File "/Users/mikekinney/python/mkinney_pyinfra/venv/lib/python3.7/site-packages/python_dateutil-2.8.1-py3.7.egg/dateutil/parser/_parser.py", line 1374, in parse
    return DEFAULTPARSER.parse(timestr, **kwargs)
  File "/Users/mikekinney/python/mkinney_pyinfra/venv/lib/python3.7/site-packages/python_dateutil-2.8.1-py3.7.egg/dateutil/parser/_parser.py", line 649, in parse
    raise ParserError("Unknown string format: %s", timestr)

    [pyinfra_cli.exceptions] dateutil.parser._parser.ParserError: Unknown string format: %date%-%time%
```

Sorry. Looks like a bug I introduced.

[pyinfra-debug.log](https://github.com/Fizzadar/pyinfra/files/4233528/pyinfra-debug.log)

**Comment by Fizzadar** [comment]
So this looks like it's because we're not respecting `shell_executable` in docker; this should be fixed in https://github.com/Fizzadar/pyinfra/commit/6ed62326167a900de22fe1aca431434b4b91d59d!

**Comment by Fizzadar** [comment]
This is now fixed/released in `v0.14.2`!
